web/client/codechecker_client/cmd_line_client.py

```diff
@@ -873,6 +904,7 @@ def report_to_html(client, reports, output_dir):
                 checked_file, output_path))
 
         html_builder.create_index_html(output_dir)
+        print_stats(reports, file_stats, severity_stats)
```

Comment:
Wouldn't be better to generate summary.html for the html output?

Reply:
We already implemented this feature. See #2036.

Reply:
If we generate a html summary too, I do not see the extra benefit of printing it here too, but ok.